motor_monitor_and_vibration/README.md

```diff
@@ -0,0 +1,186 @@
+  - [Create a Route with JSONata](#create-a-route-with-jsonata)
+  - [Configure Vibration Alerts](#configure-vibration-alerts)
+  - [Blues Wireless Community](#blues-wireless-community)
+
+
+## Overview
+
+The Motor Monitor and Vibration app monitors a motor's activity (on or off) and vibration and optionally
+send alerts when the vibration is not within the expected range for the motor's current activity.
+
+
+## Hardware you will need
+
+* AC to USB adapter (power brick) with a compatible micro USB cable
+* (optional - for power outage events) [LiPo battery with JST cable](https://shop.blues.io/products/5-000-mah-lipo-battery)
+* [Notecarrier-B](https://shop.blues.io/products/carr-b)
+* [Notecard](https://blues.io/products/notecard/)
+* A switched motor that has a 3v3 input or output that controls or indicates activity
+* A pair of jumper cables
+
+
+## Setup overview
+
+With the switched motor (such as a motor and relay) already setup and ready, you first connect the Notecarrier to a computer to configure the Notecard. Once that is complete, the Notecarrier is disconnected from the computer and is securely mounted into the motor, with signalling connections made to the Notecarrier from the motor and the computer's USB connection to the Notecarrier is replaced with a power brick.
+
+## Hardware Initial Setup
+
+This initial setup configures the Notecard to provide activity and vibration monitoring.
+
+1. Assemble the Notecard and Notecarrier as described [here](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/).
+2. Connect the Notecarrier to your PC with the micro USB cable.
+3. (Keep the Notecard connected to the computer with the in-browser terminal active until you have completed [Notecard Setup](#notecard-setup).)
+
+## Cloud Setup
+
+Sign up for a free account on [Notehub.io](https://notehub.io) and create a new project.
+
+## Notecard Setup
+
+In these steps, you will use the in-browser terminal to configure your Notecard to report the motor state and motion events to your Notehub project. You may also skip these individual steps and use the [#Notecard-CLI] to configure the Notecard using a single command.
+
+### Firmware
+
+The Notecard should use [firmware version 3.5.1](https://dev.blues.io/notecard/notecard-firmware-updates/#v3-5-1-october-7th-2022) or higher version. The simplest way to update firmware is to do an [over-the-air update](https://dev.blues.io/notecard/notecard-firmware-updates/#ota-dfu-with-notehub).
+
+### Configure Notehub Project and Connection Mode
+
+Set the ProductUID for the Notecard by pasting the command below into the in-browser terminal. Make sure to replace `com.your-company:your-product-name` with the ProductUID from your Notehub project, which can be found below your project's name in the dashboard at https://notehub.io. Also, replace the placeholder serial number (`"sn:" "motor-location"`) with the location of your fridge (e.g. "kitchen").
```

Comment:
Serial number part needs updating. Looking like it's the snippet from the fridge monitor's README.md.